src/main/java/org/apache/sling/scripting/jsp/JspScriptEngineFactory.java

```diff
@@ -272,7 +272,8 @@ private void callErrorPageJsp(final Bindings bindings,
         if (io == null || jspfh == null) {
             logger.warn("callJsp: JSP Script Engine seems to be shut down concurrently; not calling {}",
                     scriptHelper.getScript().getScriptResource().getPath());
-            return;
+            throw new Exception("callJsp: JSP Script Engine seems to be shut down concurrently; not calling {"+
```

Comment:
You don't need { and } here anymore, the placeholder were replaced by the logging engine in the previous form.

Reply:
@rombert  true , I missed that . will remove . Thanks